### ISSUE TITLE
Second mc_line move required after probing completes

### ISSUE THREAD

**Issue by nickw89509** [McNugget6750/K40-grbl-board#3]
Reviewed your changes the that you commented out on Line 355 of motion_control.c is need to do the Z retraction after the probing is completed

**Comment by McNugget6750** [comment]
Good point! Let me check! BTW: I moved grbl to a different location! Please check this out and post issues here: https://github.com/McNugget6750/grbl

**Comment by McNugget6750** [comment]
Since the mentioned section was completely gone in the new 0.9j master I'm not sure If they handle that differently now. But my understanding of grbl is quite limited...

**Comment by McNugget6750** [comment]
Running a test right now. I only have two axis to test with so I don't know if having the code section or not makes any difference...
